react-frontend/src/App.js

```diff
@@ -1,24 +1,12 @@
 import React from "react";
-import logo from "./logo.svg";
 import "./App.css";
+import "./components/MainComponent";
```

Comment:
We need to use "absolute imports". i.e. start your import path from directory in "src" without a "." or ".." prefix.

e.g. `import "components/MainComponent";`